webpack.config.prod.js

```diff
@@ -4,7 +4,6 @@ const webpack = require('webpack');
 const baseConfig = require('./webpack.config');
 
 module.exports = Object.assign(baseConfig, {
-  devtool: 'none',
```

Comment:
I would keep this one as it was. The one in the webpack.config.js  we can change to `eval-cheap-source-map`